go.mod

```diff
@@ -231,6 +232,7 @@ replace (
 	github.com/exponent-io/jsonpath => github.com/exponent-io/jsonpath v0.0.0-20151013193312-d6023ce2651d
 	github.com/fatih/camelcase => github.com/fatih/camelcase v1.0.0
 	github.com/felixge/httpsnoop => github.com/felixge/httpsnoop v1.0.1
+	github.com/flowstack/go-jsonschema => github.com/flowstack/go-jsonschema v0.1.1
```

Comment:
what is pulling this in? this and the imports below (github.com/xeipuuv/gojson...) looks like we're importing another json schema stack, which we really don't want... we already have one in kube-openapi and github.com/go-openapi

Reply:
it looks like this is only present because it's referenced from a test in https://github.com/google/gnostic/blob/master/cmd/protoc-gen-jsonschema/plugin_test.go

we don't want this dependency or the transitive dependencies that come with it... please file an issue with the gnostic library to drop this before we pick this up

Reply:
@apelisse brought this up as well, gnostic introduced a protoc-gen-jsonschema tool (https://github.com/google/gnostic/commit/dd0d92fd224dee6096d9f9c19bbb69cb88c57b61). We don't depend on it on our side as we use gnostic strictly for the openapiv2 and v3 proto models.

Reply:
(and revert kube-openapi to googleapis/gnostic if that cannot happen quickly, so we don't have kube-openapi versions we can't pick up in k8s.io/kubernetes)

Reply:
it doesn't matter whether we link to it or not, it still pollutes our dependency graph

Reply:
We need to upgrade gnostic in this k/k release to pick up an OpenAPI proto bug fix. I understand that this does add unwanted dependencies to our dependency graph, but do we have an alternative?

I'm using this PR to rename the library so the next diff won't be too large.